test/extended/image_ecosystem/plugin.go

```diff
@@ -412,20 +418,49 @@ var _ = g.Describe("[image_ecosystem][jenkins][Slow] openshift pipeline plugin",
 		time.Sleep(10 * time.Second) // Give project time to initialize
 
 		g.By("kick off the build for the jenkins ephermeral and application templates")
-		tag := []string{"openshift/jenkins-plugin-snapshot-test:latest"}
+		tag := []string{localPluginSnapshotImage}
 		hexIDs, err := exutil.DumpAndReturnTagging(tag)
-		var jenkinsEphemeralPath string
-		var testingSnapshot bool
-		if len(hexIDs) > 0 && err == nil {
-			// found an openshift pipeline plugin test image, must be testing a proposed change to the plugin
-			jenkinsEphemeralPath = exutil.FixturePath("testdata", "jenkins-ephemeral-template-test-new-plugin.json")
-			testingSnapshot = true
+
+		// If the user has expressed an interest in local plugin testing by setting the
+		// SNAPSHOT_JENKINS_IMAGE environment variable, try to use the local image. Inform them
+		// either about which image is being used in case their test fails.
+		snapshotImagePresent := len(hexIDs) > 0 && err == nil
+		useSnapshotImage := os.Getenv(useLocalPluginSnapshotEnvVarName) != "" && snapshotImagePresent
+
+		//TODO disabling oauth until we can update getAdminPassword path to handle oauth (perhaps borrow from oauth integration tests)
+		newAppArgs := []string{exutil.FixturePath("..", "..", "examples", "jenkins", "jenkins-ephemeral-template.json"), "-p", "ENABLE_OAUTH=false"}
+
+		if useSnapshotImage {
+			ginkgolog("")
+			ginkgolog("")
+			ginkgolog("IMPORTANT: You are testing a local jenkins snapshot image.")
+			ginkgolog("In order to target the official image stream, you must unset %s before running extended tests.", useLocalPluginSnapshotEnvVarName)
+			ginkgolog("")
+			ginkgolog("")
+
+			// Create an imagestream based on the Jenkins' plugin PR-Testing image (https://github.com/openshift/jenkins-plugin/blob/master/PR-Testing/README).
+			snapshotImageStream := "jenkins-plugin-snapshot-test"
+			err = oc.Run("new-build").Args("-D", fmt.Sprintf("FROM %s", localPluginSnapshotImage), "--to", snapshotImageStream).Execute()
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			err = oc.Run("logs").Args("-f", "bc/jenkins-plugin-snapshot-test").Execute()
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			// Supplant the normal imagestream with the local imagestream using template parameters
+			newAppArgs = append(newAppArgs, "-p", fmt.Sprintf("NAMESPACE=%s,JENKINS_IMAGE_STREAM_TAG=%s:latest", oc.Namespace(), snapshotImageStream))
```

Comment:
needs ENABLE_OAUTH=false
also split these parameters into multiple -p args, because we're disabling the comma separated key/value pairs behavior: https://github.com/openshift/origin/pull/11539